micro-rdk/src/native/entry.rs

```diff
@@ -76,7 +76,7 @@ pub async fn serve_web_inner(
         );
 
         let (cfg_response, cfg_received_datetime) =
-            client.get_config(network.get_ip()).await.unwrap();
+            client.get_config(Some(network.get_ip())).await.unwrap();
```

Comment:
Commenting here because I can't elsewhere, but I'd expect to see a ConfigMonitor added for native as well. Is there a reason one was not?

Reply:
I wanted to get one working before I added it to another : ) will add in my next commit!